utils/vpr_pbtype_to_eblif.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+"""
+Generate an eblif file targetting a given pb_type.xml file.
+Used for testing a leaf pb_type can be placed and routed with Verilog to
+Routing.
+
+Primarily used by the vpr_test_pbtype cmake function (which is automatically part
+of v2x_test_both cmake function).
+"""
+
+import argparse
+import math
+import os
+import os.path
+import subprocess
+import sys
+import tempfile
+
+from typing import List
+
+import lxml.etree as ET
+
+from lib.flatten import flatten
+from lib.pb_type import Port, ports, find_leaf
+
+
+def blif(name: str, inputs: List[Port], outputs: List[Port]) -> str:
+    """
```

Comment:
Please actually write a docstring here.  doctests are not complete documentation

Reply:
Done.

Reply:
Done.